tests/ctl/nmstatectl_test.py

```diff
@@ -57,20 +69,38 @@ def test_run_ctl_directly_show_empty():
     nmstatectl.main()
 
 
-@mock.patch('sys.argv', ['nmstatectl', 'show', 'eth1'])
+@mock.patch('sys.argv', ['nmstatectl', 'show', 'non_existing'])
```

Comment:
`non_existing_iface`?

Reply:
changed to `non_existing_interface` to mach integration tests.